components/SmiteGodCards.tsx

```diff
@@ -19,40 +19,52 @@ export const SmiteGodCards = () => {
   );
 
   return (
-    <Box
-      display="flex"
-      flex={1}
-      justifyContent="center"
-      marginTop={5}
-      flexWrap="wrap"
-      gap={5}
-      height="fit-content"
-    >
-      {randomGods.map((god) => (
-        <SmiteGodCard
-          key={god.id}
-          sx={{
-            backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${god.godCard_URL})`,
-          }}
-        >
-          <CardContent>
-            <Typography
-              gutterBottom
-              variant="h4"
-              color="white"
-              fontWeight={700}
-            >
-              {god.Name}
-            </Typography>
-            <Typography color="white" fontWeight={700}>
-              {god.Pantheon}
-            </Typography>
-            <Typography color="white" fontWeight={700}>
-              {god.Title}
-            </Typography>
-          </CardContent>
-        </SmiteGodCard>
-      ))}
-    </Box>
+    randomGods.length ? (
+      <Box
+        display="flex"
+        flex={1}
+        justifyContent="center"
+        marginTop={5}
+        flexWrap="wrap"
+        gap={5}
+        height="fit-content"
+      >
+        {randomGods.map((god) => (
+          <SmiteGodCard
+            key={god.id}
+            sx={{
+              backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${god.godCard_URL})`,
+            }}
+          >
+            <CardContent>
+              <Typography
+                gutterBottom
+                variant="h4"
+                color="white"
+                fontWeight={700}
+              >
+                {god.Name}
+              </Typography>
+              <Typography color="white" fontWeight={700}>
+                {god.Pantheon}
+              </Typography>
+              <Typography color="white" fontWeight={700}>
+                {god.Title}
+              </Typography>
+            </CardContent>
+          </SmiteGodCard>
+        ))}
+      </Box>
+    ) : (
+      <Box
+        position="absolute"
+        top="50%"
+        left="50%"
```

Comment:
We should try to use Flexbox instead of setting the position. This causes an issue on mobile where the text is not aligned in the center.
![image](https://github.com/hphothong/smite/assets/24667860/401210e6-7c4f-4503-82cd-6a96bc14283c)